### DISCUSSION TITLE
DaisyUI should use the APCA algorithm

### DISCUSSION THREAD

**Discussion by kucho** [saadeghi/daisyui#2494] [Ideas / feature requests]
Now that DaisyUI is using OKLCH, the color contrast should be calculated using APCA instead.

I was trying to create a color theme using the [Harmony Palette](https://www.figma.com/community/file/1287828769207775946), the current algorithm is not using the best color for foreground text.

For instance, the colour `Cyan.600` should have a white text.

**Comment by saadeghi** [comment]
I agree.  
Hopefully `culori` package adds the support because we're currently using that API and I don't want to add a new dependcy just for APCA.

I'm following this issue and as soon as they add the API I'm going to update the function:  
https://github.com/Evercoder/culori/issues/177
